packages/server/src/models/Session.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (sequelize, DataTypes) => {
+  const Session = sequelize.define('Session',
+    {
+      sid: {
+        unique: true,
+        primaryKey: true,
+        type: DataTypes.STRING,
+        validate: {
+          notEmpty: true,
+        }
+      },
+      modelId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        }
+      },
+      modelType: {
```

Comment:
Not sure I understand the purpose of `modelId` and `modelType`—how are we using them?

Reply:
Currently, we only have admin sessions, but we will have also users sessions. The idea is that we can track them in the same table but having a polymorphic relation for each of them

Reply:
Wondering—would an `admin` boolean column suffice or would it be awkward?

Reply:
I could work, will check if there is a way to define polymorphic relations with a boolean